tests/run/public_fused_types.srctree

```diff
@@ -227,7 +227,7 @@ ae(myobj.cpdef_method[cy.int, cy.float](10, 10.0), (10, 10.0))
 
 d = {'obj': obj, 'myobj': myobj, 'ae': ae}
 
-exec(s, d)
```

Comment:
The last pending failing test caused directly by this PR. This needs to be fixed and uncommented before considering this PR as ready.

Reply:
This is fixed by commit https://github.com/cython/cython/pull/5931/commits/2c39b77ae75aee573eccb287fe70bc266c6f8b77